### ISSUE TITLE
some users causing bot to crash

### ISSUE THREAD

**Issue by MSFTserver** [lbryio/lbry-wunderbot#118]
when some users use commands the bot trips out and crashes itself, so these users cant use commands

```
1|bot      | TypeError: Cannot read property 'roles' of null
1|bot      |     at exports.hasPerms (/home/lbry/lbry-wunder/bot/helpers.js:11:21)
1|bot      |     at Client.bot.on.msg (/home/lbry/lbry-wunder/bot/modules/spam-detection.js:35:7)
1|bot      |     at emitOne (events.js:121:20)
1|bot      |     at Client.emit (events.js:211:7)
1|bot      |     at MessageCreateHandler.handle (/home/lbry/lbry-wunder/node_modules/discord.js/src/client/websocket/packets/handlers/MessageCreate.js:9:34)
1|bot      |     at WebSocketPacketManager.handle (/home/lbry/lbry-wunder/node_modules/discord.js/src/client/websocket/packets/WebSocketPacketManager.js:103:65)
1|bot      |     at WebSocketConnection.onPacket (/home/lbry/lbry-wunder/node_modules/discord.js/src/client/websocket/WebSocketConnection.js:330:35)
1|bot      |     at WebSocketConnection.onMessage (/home/lbry/lbry-wunder/node_modules/discord.js/src/client/websocket/WebSocketConnection.js:293:17)
1|bot      |     at WebSocket.onMessage (/home/lbry/lbry-wunder/node_modules/ws/lib/EventTarget.js:99:16)
1|bot      |     at emitOne (events.js:116:13)
1|bot      | WARNING: NODE_APP_INSTANCE value of '0' did not match any instance config file names.
1|bot      | WARNING: See https://github.com/lorenwest/node-config/wiki/Strict-Mode
```
```
1|bot      | treating !stats from UserID:<@224345489498308608> || UserName: kcSeb as command
1|bot      | TypeError: Cannot read property 'roles' of null
1|bot      |     at exports.hasPerms (/home/lbry/lbry-wunder/bot/helpers.js:11:21)
1|bot      |     at Client.bot.on.msg (/home/lbry/lbry-wunder/bot/modules/spam-detection.js:35:7)
1|bot      |     at emitOne (events.js:121:20)
1|bot      |     at Client.emit (events.js:211:7)
1|bot      |     at MessageCreateHandler.handle (/home/lbry/lbry-wunder/node_modules/discord.js/src/client/websocket/packets/handlers/MessageCreate.js:9:34)
1|bot      |     at WebSocketPacketManager.handle (/home/lbry/lbry-wunder/node_modules/discord.js/src/client/websocket/packets/WebSocketPacketManager.js:103:65)
1|bot      |     at WebSocketConnection.onPacket (/home/lbry/lbry-wunder/node_modules/discord.js/src/client/websocket/WebSocketConnection.js:330:35)
1|bot      |     at WebSocketConnection.onMessage (/home/lbry/lbry-wunder/node_modules/discord.js/src/client/websocket/WebSocketConnection.js:293:17)
1|bot      |     at WebSocket.onMessage (/home/lbry/lbry-wunder/node_modules/ws/lib/EventTarget.js:99:16)
1|bot      |     at emitOne (events.js:116:13)
1|bot      | WARNING: NODE_APP_INSTANCE value of '0' did not match any instance config file names.
1|bot      | WARNING: See https://github.com/lorenwest/node-config/wiki/Strict-Mode
```

**Comment by MSFTserver** [comment]
```
// FETCH a member. Useful if an invisible user sends a message. message.guild.fetchMember(message.author) .then(member => { // The member is available here. });
```

**Comment by MSFTserver** [comment]
possible fix

**Comment by nikooo777** [comment]
can you try to include it? Sounds like a clean workaround!

**Comment by MSFTserver** [comment]
yea I'll see about adding that

**Comment by Coolguy3289** [comment]
Fixed in #149
